IOS/SwiftSeedProject/SwiftSeedProject/Interfaces/Components/NewsCollection/Cells/SSNewsCollectionCellVM.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  SSNewsCollectionCellVM.swift
+//  SwiftSeedProject
+//
+//  Created by Lucas Pelizza on 4/18/17.
+//  Copyright © 2017 Making Sense. All rights reserved.
+//
+
+import Foundation
+
+final class SSNewsCollectionCellVM {
```

Comment:
Also, `NewsCollectionCellVM` doesn't feel like an appropriate name for a `ViewModel`, it's tied to UI.

For what I can see here, this ViewModel represents a piece of news, and 'news' is a tricky word (an irregular plural), so maybe we can start using the name 'Article', and this would be an `ArticleViewModel`or `NewsArticleViewModel`